core/themes/basis/template.php

```diff
@@ -44,7 +44,22 @@ function basis_preprocess_layout(&$variables) {
 }
 
 /**
- * Implements template_preprocess_header().
+ * Prepares variables for node templates.
+ *
+ * @see node.tpl.php
+ */
+function basis_preprocess_node(&$variables) {
+  if ($variables['status'] == NODE_NOT_PUBLISHED) {
+    $name = node_type_get_name($variables['type']);
+    $variables['title_suffix'] = array(
+      '#type' => 'markup',
+      '#markup' => '<div class="unpublished-indicator">' . t('This !type is unpublished.', array('!type' => $name)) . '</div>',
```

Comment:
`node_type_get_name()` returns the node type label, which may include characters like `<` and `>`. This replacement needs to be a sanitized version (use `@type` instead of `!type`). It should also be translated through `t()`.